modules/tinymce/CHANGELOG.md

```diff
@@ -110,6 +110,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Anchor elements would render incorrectly when using the `allow_html_in_named_anchor` option #TINY-3799
 - The `AstNode` HTML serializer did not serialize `pre` or `textarea` elements correctly when they contained newlines #TINY-8446
 - Fixed sub-menu items not read by screen readers. Patch contributed by westonkd #TINY-8417
+- Drag and drop would not properly detect images and respect the `paste_data_images` option in some cases #TINY-8486
```

Comment:
Was this broken in 5.x? I believe this is something that got broken in the 6.0 work, so in that case we shouldn't have a changelog.

Reply:
For reference here's the old logic: https://github.com/tinymce/tinymce/blob/master/modules/tinymce/src/plugins/paste/main/ts/core/DragDrop.ts#L47. As you can see it didn't filter on images at all, which was something we tried to fix IIRC. So I'm pretty sure this wasn't a bug in 5.x.